<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>115</width>
     <height>114</height>
    </rect>
   </property>
   <property name="title">
    <string>菜单</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>西红柿炒鸡蛋</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="text">
       <string>青椒肉丝</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="text">
       <string>大盘鸡</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_4">
      <property name="text">
       <string>红烧牛腩</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>50</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>奥迪</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>奔驰</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>宝马</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择车型</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>30</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>英雄名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>英雄属性</string>
    </property>
   </column>
   <item>
    <property name="text">
     <string>德玛西亚之力</string>
    </property>
    <item>
     <property name="text">
      <string>盖伦</string>
     </property>
     <property name="text">
      <string>战士</string>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>小法</string>
    </property>
    <item>
     <property name="text">
      <string>维嘉</string>
     </property>
     <property name="text">
      <string>法师</string>
     </property>
    </item>
   </item>
  </widget>
  <widget class="MyWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>611</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>460</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>设置进度条为一半</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyWidget</class>
   <extends>QWidget</extends>
   <header location="global">mywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
